### PR TITLE
Prep for rec-4.4.0-beta1 release

### PR DESCRIPTION
**Pull request by omoerbeek**
### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [ ] tested this code
- [X] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by omoerbeek** [comment]
https://powerdnsblog.wordpress.com/?p=3166&preview=true

File: pdns/recursordist/docs/changelog/4.4.rst [hunk 1]
```diff
@@ -1,5 +1,96 @@
 Changelogs for 4.4.x
 ====================
+.. changelog::
+  :version: 4.4.0-beta1
+  :released: 31th of August 2020
```
Comment (rgacogne) [inline pdns/recursordist/docs/changelog/4.4.rst:5]:
```suggestion
  :released: 31st of August 2020
```

**Review by rgacogne — APPROVED** [review]
One nit!

File: pdns/recursordist/docs/changelog/4.4.rst [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+    :tags: Improvements
+    :pullreq: 9391
+
+    Export record cache lock (contention) stats via the various channels.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9396
+
+    Allow multipe local data records when doing RPZ IP matching.
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.4.rst:49]:
```suggestion
    Allow multiple local data records when doing RPZ IP matching.
```

File: pdns/recursordist/docs/changelog/4.4.rst [hunk 3]
```diff
@@ -1,5 +1,96 @@
...
+
+  .. change::
+    :tags: Internals, Bug Fixes
+    :pullreq: 9404
+
+    Test-recursor: remove duplicate cd directive (Josh Soref).
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.4.rst:55]: this can be skipped

File: pdns/recursordist/docs/changelog/4.4.rst [hunk 4]
```diff
@@ -1,5 +1,96 @@
...
+    :pullreq: 9351
+    :tickets: 9227
+
+    If we have an NS in cache, use it in the forwarder case.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9371
+
+    Update netmaskgroup.rst (phonedph1).
```
Comment (Habbie) [inline pdns/recursordist/docs/changelog/4.4.rst:74]: skip

**Review by Habbie — CHANGES_REQUESTED** [review]

**Pull request merged**